napari/utils/_tests/test_theme.py

```diff
@@ -30,6 +31,13 @@ def test_get_theme():
     assert isinstance(theme, Theme)
 
 
+def test_get_system_theme(monkeypatch):
+    monkeypatch.setattr('napari.utils.theme.get_system_theme', lambda: 'light')
+    theme = get_theme('system', False)
```

Comment:
Optional, but would make this test more readable (i.e. I don't need to go look at the definition of `get_theme` to understand what the `False` is for.

```suggestion
    theme = get_theme('system', as_dict=False)
```